.github/workflows/gpu.yml

```diff
@@ -17,11 +17,10 @@ jobs:
 
     strategy:
       matrix:
-        os: ['ubuntu-22.04']
         build_type: [Release]
         compiler: [{c: gcc, cxx: g++}]
         shared_library: ['ON', 'OFF']
-    runs-on: level_zero
+    runs-on:  [ "DSS-LEVEL_ZERO", "DSS-UBUNTU" ]
```

Comment:
so this means that both labels are required to run this workflow, right?

Reply:
I hope so :wink: I'll check it yet

Reply:
ok, it worked perfectly: https://github.com/oneapi-src/unified-memory-framework/actions/runs/8251864728/job/22597511622#step:1:2